.github/workflows/docker.yml

```diff
@@ -39,12 +39,10 @@ jobs:
         id: set-matrix
         if: steps.filter.outputs.docker_toml == 'true' || github.event_name == 'push'
         run: |
-          cd ./docker/
-          ls
-          dockerfiles=$(ls Dockerfile_* | jq -R -s -c 'split("\n")[:-1]')
+          dockerfiles=$(find docker -type f -name 'Dockerfile*' -exec basename {} \; | jq -R -s -c 'split("\n")[:-1]')
           echo "dockerfiles: $dockerfiles"
           echo "dockerfiles=$dockerfiles" >> $GITHUB_OUTPUT
-          imagenames=$(ls Dockerfile_* | sed -e 's/Dockerfile_/ghcr.io\/ptb-mr\/mrpro_/' | jq -R -s -c 'split("\n")[:-1]')
+          imagenames=$(find docker -type f -name 'Dockerfile*' -exec basename {} \; | sed 's/^Dockerfile/ghcr.io\/ptb-mr\/mrpro/' | jq -R -s -c 'split("\n")[:-1]')
```

Comment:
It's a little bit messy with the folders and filenames. I believe a better way is to change the folder in the single place. Now we take only the filenames here and append the `docker` folder prefix later.

A better way would mean that we don't use `basename` here and including the whole path to the `imagenames` and then get away from `docker` folder in all other steps.

I think it's obviously has advantages that if the change the folder some day we would need to change the related folder in the single line and not in each following step.